site/src/xServices/workspace/workspaceXService.ts

```diff
@@ -56,7 +57,7 @@ export interface WorkspaceContext {
   workspace?: TypesGen.Workspace
   template?: TypesGen.Template
   build?: TypesGen.WorkspaceBuild
-  getWorkspaceError?: Error | unknown
+  getWorkspaceError?: AxiosError
```

Comment:
Usually, errors are kinda tricky to identify because they can be anything. But I think it is ok for now we assume those errors from the API will be mostly AxiosError but just a heads up.